### PR TITLE
About page & related

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/aredotna/ervell/issues/780

Opening this to knock down some tiny remaining issues with it (mobile, fix a few hrefs)

### For deploy

- [x] Fix remaining hrefs
- [x] Smooth scroll
- [x] Mobile

### Post-deploy

- [ ] Inline FAQ

about

### CODE REVIEW

**Comment by dzucconi** [comment]
This includes a fairly large amount of housekeeping.

File: apps/home/routes.coffee [hunk 1]
```diff
@@ -21,12 +21,16 @@ cached = require '../../lib/cached'
         #{require '../../components/block_v2/queries/block'}
       """
 
-    GQL send
+  .catch (->) # Ignore query errors
 
-  .then ({ channel: { blocks }}) ->
-    res.locals.sd.DEMO_BLOCKS = demoBlocks = shuffle blocks
+  .then (response) ->
```
Comment (dzucconi) [inline apps/home/routes.coffee]: I hate that native promises don't have an `always`. So `catch` followed by another `then`

File: components/ui/variables.styl [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @require './buttons/variables'
 @require './colors/variables'
 @require './forms/variables'
+@require './layout/variables'
 @require './typography/variables'
```
Comment (dzucconi) [inline components/ui/variables.styl]: This is a setup for a reworking of the channel block grid CSS that I have cooking.

File: components/layout/legacy_variables.styl [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+s_feed = 23px
+
+// TODO: Delete these as they are eliminated
+// units
+unit = 20px
+s_margin = (unit / 2)
+block_width = 315px
+block_width_mar = block_width + (s_margin * 2)
+topbar_h = 50px
+container_pad = 20px
```
Comment (dzucconi) [inline components/layout/legacy_variables.styl]: There's not a huge amount of use for these. The pattern I've been using is UI components have their own variable files which then get required at the top of everything. So you get local isolation but can use them anywhere.

**Comment by dzucconi** [comment]
Also I realize the issue says same scroll speed as homepage. I had just remembered there was no scroll speed when we talked about it upstate. Will do the smoothscroll.

**Comment by dzucconi** [comment]
@broskoski Done here. Staging?

File: components/ui/variables.styl (hunk 2 above)
Comment (broskoski) [inline components/ui/variables.styl]: 🍜

File: components/layout/legacy_variables.styl (hunk 3 above)
Comment (broskoski) [inline components/layout/legacy_variables.styl]: Yeah that is nicer

**Comment by broskoski** [comment]
Yes, looks good to me, lets see it on staging

**Pull request merged**